### PR TITLE
Update recipe to not use Python

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
It seems the cmake still looks for python

```
-- Found PythonInterp: /Users/travis/miniconda3/bin/python (found version "3.6.5") 
```

and one more thing, the lib folder was not created on apple

```
cp: directory /Users/travis/miniconda3/conda-bld/gtest_1528093677658/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib does not exist
```

Is there a ``make install`` after you ran the ``cmake`` ?

**Comment by SylvainCorlay** [comment]
Yes, I think that there should be a make install, but I merely wanted to move to conda-build 3 in this PR.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,12 +26,6 @@ requirements:
     - cmake
```
Comment (scopatz) [inline recipe/meta.yaml:26]: Seems like Python is still in the build reqs.

Comment (SylvainCorlay) [reply recipe/meta.yaml:26]: Yep, just re-added as per earlier comment.

**Comment by marcelotrevisani** [comment]
It's trying to install the library into the system 

```
CMake Error at cmake_install.cmake:41 (file):
  file INSTALL cannot copy file
  "/home/conda/feedstock_root/build_artifacts/gtest_1528114550454/work/build_static/libgtest.a"
  to "/usr/local/lib/libgtest.a".
```

Probably you have to set some variables in ``cmake``, like:
```
cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} .
```

File: recipe/build.sh [hunk 2]
```diff
@@ -3,22 +3,18 @@
...
 make
-cp libgtest.a $PREFIX/lib/
-cp libgtest_main.a $PREFIX/lib/
+make install
 cd $GTEST_DIR
 
 # Build and copy dynamic libraries
 mkdir build_dynamic
 cd build_dynamic
 cmake $GTEST_DIR -Dgtest_build_tests=ON
```
Comment (marcelotrevisani) [inline recipe/build.sh]: I think you also have to set another variables like:

```
cmake $GTEST_DIR -Dgtest_build_tests=ON -DCMAKE_INSTALL_DIR=${PREFIX}
```

Comment (SylvainCorlay) [reply recipe/build.sh]: Yes, `make install` works (if you set the `CMAKE_INSTALL_DIR=${PREFIX}` obviously).

It seems that this version of the build.sh comes from an early version of gtest where the make install was not set up correctly. Pushing a fixed version.

**Comment by scopatz** [comment]
This didn't bump the build number so a new package will not be pushed, FYI

**Comment by SylvainCorlay** [comment]
> This didn't bump the build number so a new package will not be pushed, FYI

I know but this build number was never pushed on OS X and Linux.

**Comment by scopatz** [comment]
AHh ok!

**Pull request merged**